ush/machine/hopper.yaml

```diff
@@ -0,0 +1,53 @@
+platform:
+  WORKFLOW_MANAGER: rocoto
+  NCORES_PER_NODE: 40
+  SCHED: slurm
+  CCPA_OBS_DIR: /scratch2/BMC/det/UFS_SRW_App/develop/obs_data/ccpa/proc
+  MRMS_OBS_DIR: /scratch2/BMC/det/UFS_SRW_App/develop/obs_data/mrms/proc
+  NDAS_OBS_DIR: /scratch2/BMC/det/UFS_SRW_App/develop/obs_data/ndas/proc
+  METPLUS_PATH: /contrib/METplus/METplus-4.1.1
+  MET_BIN_EXEC: bin
+  MET_INSTALL_DIR: /contrib/met/10.1.1
+  DOMAIN_PREGEN_BASEDIR: /scratch2/BMC/det/UFS_SRW_App/develop/FV3LAM_pregen
+  PARTITION_DEFAULT: hera
+  QUEUE_DEFAULT: batch
+  PARTITION_FCST: hera
+  QUEUE_FCST: batch
+  PARTITION_HPSS: service
+  QUEUE_HPSS: batch
+  RUN_CMD_FCST: srun --export=ALL
+  RUN_CMD_POST: srun --export=ALL
+  RUN_CMD_SERIAL: time
+  RUN_CMD_UTILS: srun --export=ALL
+  RUN_CMD_AQM: srun -n ${nprocs} --export=ALL
+  RUN_CMD_AQMLBC: srun --export=ALL -n ${NUMTS}
+  SCHED_NATIVE_CMD: --export=NONE
+  PRE_TASK_CMDS: '{ ulimit -s unlimited; ulimit -a; }'
+  TEST_EXTRN_MDL_SOURCE_BASEDIR: /scratch2/BMC/det/UFS_SRW_App/develop/input_model_data
+  TEST_PREGEN_BASEDIR: /scratch2/BMC/det/UFS_SRW_App/develop/FV3LAM_pregen
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_ICS: /scratch2/BMC/det/UFS_SRW_app/develop/dummy_FV3GFS_sys_dir
+  TEST_ALT_EXTRN_MDL_SYSBASEDIR_LBCS: /scratch2/BMC/det/UFS_SRW_app/develop/dummy_FV3GFS_sys_dir
+  TEST_VX_FCST_INPUT_BASEDIR: '/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_det/{{workflow.PREDEF_GRID_NAME}}'
+  FIXaer: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_aer
+  FIXgsm: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_am
+  FIXlut: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_lut
+  FIXorg: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_orog
+  FIXsfc: /scratch2/BMC/det/UFS_SRW_App/develop/fix/fix_sfc_climo
+  FIXshp: /scratch2/BMC/det/UFS_SRW_App/develop/NaturalEarth
+  EXTRN_MDL_DATA_STORES: hpss aws nomads
+
+data:
+  obs:
+    RAP_obs: /scratch2/BMC/public/data/grids/rap/obs
+cpl_aqm_parm:
+  AQM_CONFIG_DIR: /scratch2/NCEPDEV/naqfc/RRFS_CMAQ/aqm/epa/data
+  AQM_BIO_DIR: /scratch2/NCEPDEV/naqfc/RRFS_CMAQ/aqm/bio
+  AQM_DUST_DIR: /groups/ESS3/ytang/RRFS-input/FENGSHA
+  AQM_CANOPY_DIR: /groups/ESS3/ytang/RRFS-input/canopy
+  AQM_FIRE_DIR: /groups/ESS3/ytang/RRFS-input/emissions/GSCE/RAVE.in.793/RAVE_RT
+  AQM_LBCS_DIR: /groups/ESS3/ytang/RRFS-input/LBCS/UFSAQM_NA13km_GEOS5_LBC_v3
+  AQM_GEFS_DIR: /groups/ESS3/ytang/RRFS-input/GEFS_aerosol
+  NEXUS_INPUT_DIR: /groups/ESS3/ytang/RRFS-input/nexus_emissions
+  NEXUS_FIX_DIR: /groups/ESS3/ytang/RRFS-input/nexus/fix
+  PT_SRC_BASEDIR: 
```

Comment:
PT_SRC_BASEDIR: /groups/ESS3/ytang/RRFS-input/emissions

Reply:
@ytangnoaa Added that path.